webpack.config.js

```diff
@@ -4,7 +4,7 @@ var webpack = require('webpack')
 var HtmlWebpackPlugin = require('html-webpack-plugin')
 
 if (process.env.NODE_ENV === 'production') {
-    var publicPath = 'https://neontribe.github.io/ARCVMarket/';
+    var publicPath = './';
```

Comment:
@katjam I need to be able to build it so I can test the built version on my phone, because that's the only way I can access my copy of the site on there. Not sure why that is - ask Charlie for a more informative answer. :)